packages/qwik/src/core/v2/client/dom-container.ts

```diff
@@ -64,7 +64,7 @@ import {
   vnode_setProp,
   type VNodeJournal,
 } from './vnode';
-import { vnode_diff } from './vnode-diff';
+import { isSlotProp, vnode_diff } from './vnode-diff';
```

Comment:
are we sure we want `isSlotProp` to sit @ `vnode-diff`? 🤔

Reply:
yeah, I will think about it later

Reply:
I moved this into separate file